lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -89,11 +93,35 @@ def self.deliver(*args)
       end
 
       uri = create_automation_object(options[:instance_name], automate_attrs, create_automation_object_options(options, vmdb_object))
+      _scheme, _userinfo, _host, _port, _registry, _path, _opaque, query, _fragment = MiqAeUri.split(uri, "miqaedb")
+      args = MiqAeUri.query2hash(query)
+
+      miq_request_id = nil
```

Comment:
@DavidResende0 Can you reuse `MiqAeWorkspaceRuntime.find_miq_request_id` here?  The one that uses args.